script.js

```diff
@@ -0,0 +1,79 @@
+const $ = (selector) => document.querySelector(selector);
+const $$ = (selector) => document.querySelectorAll(selector)
+
+let currentCountries = {}
+
+const getAllCountries = async function () {
+  const response = await fetch('CountriesData.json')
+  return await response.json()
+}
```

Comment:
Save `'CountriesData.json'` into constant file 
This apply to every hardcoded string that you have